src/psyclone/tests/core/signature_test.py

```diff
@@ -0,0 +1,101 @@
+# -----------------------------------------------------------------------------
```

Comment:
That line is only covered when you use python 2. I've restructured that code, so now in case of python 3 there is an unnecessary ``str()`` of a string, but all is covered now.

Reply:
Again see comment elsewhere (bad idea to use the file changed tab).